src/entities/Product.java

```diff
@@ -0,0 +1,45 @@
+package entities;
+
+import enums.MeasurementUnit;
+
+public class Product extends BaseEntity<Long>{
+    private String name;
+    private Long buyingPrice;
```

Comment:
What is difference between buying price and selling price?

Reply:
Should price be a Double instead?

Reply:
I see now that is better to remove the buying price because a product can be a single item or a complex item like a burger, a product is not an ingredient so it wouldn't make sense for me to add those attributes.
I'll remove them and just leave a double attribute called "price"

Reply:
sg!